packages/components/src/checkbox-control/style.scss

```diff
@@ -1,3 +1,76 @@
 .components-checkbox-control__input[type="checkbox"] {
-	margin-top: 0;
+	border: 1px solid #b4b9be;
+	background: #fff;
+	color: #555;
+	clear: none;
+	cursor: pointer;
+	display: inline-block;
+	line-height: 0;
+	height: 16px;
+	margin: 0 4px 0 0;
+	outline: 0;
+	padding: 0 !important;
```

Comment:
For anyone else reviewing this, this whole green section is largely copied from http://localhost:8888/wp-admin/css/forms.css?ver=5.2.2, so it is as it says on the bin, a copy of the CSS from wp-admin. So while it would be nice to refactor away the `!important`, this is a separate task bringing these components to all of wp-admin.